source/basic.tex

```diff
@@ -4146,7 +4146,7 @@
 a \term{cv-unqualified type}. Each type which is a
 cv-unqualified complete or incomplete object type or is
 \tcode{void}~(\ref{basic.types}) has three corresponding cv-qualified
-versions of its type: a \grammarterm{const-qualified} version, a
+versions of its type: a \term{const-qualified} version, a
```

Comment:
We don't have an index entry for "const-qualified".
All the other terms defined in that same sentence are just \term{volatile-qualified}, for example, so I just wanted to fix this particular bump.
I suggest to go forward with this (it's an obvious improvement).